src/stylesheets/main.less

```diff
@@ -1,370 +1,43 @@
 @import "reset.less";
```

Comment:
I think we should put all imports under the sk-container scope (maybe not mixins) to make sure we don't interfere with the user's website styling.